test-app/src/main/java/org/readium/r2/testapp/reader/ReaderRepository.kt

```diff
@@ -118,7 +119,7 @@ class ReaderRepository(
         val readerInitData = when {
             publication.conformsTo(Publication.Profile.AUDIOBOOK) ->
                 openAudio(bookId, publication, initialLocator)
-            publication.conformsTo(Publication.Profile.EPUB) ->
+            publication.conformsTo(Publication.Profile.EPUB) || publication.readingOrder.allAreHtml ->
```

Comment:
To support Readium Web Publication with HTML resources such as https://readium.org/webpub-manifest/examples/MobyDick/manifest.json